dist/playkit-youbora.js.map

```diff
@@ -0,0 +1 @@
index) + '/' + res\n          }\n          prevParent = parentResource.slice(0, index) + '/'\n        }\n\n        if (matches[3] === '.m3u8' || matches[3] === '.m3u') { // It is m3u8 or m3u...\n          if (!lastManifest) {\n            var request = new YBRequest(res, null, null, { cache: true })\n\n            request.on(YBRequest.Event.SUCCESS, function (resp) {\n              this._parseWithManifest(resp, prevParent, matches)\n            }.bind(this))\n\n            request.on(YBRequest.Event.ERROR, function (resp) {\n              this.done()\n            }.bind(this))\n\n            request.send()\n          } else {\n            this._parseWithManifest(lastManifest, prevParent, matches)\n          }\n        } else { // It is mp4 or ts...\n          switch (matches[3]) {\n            case '.mp4':\n            case '.m4s':\n              this._transportFormat = TransportFormat.MP4\n              break\n            case '.ts':\n              this._transportFormat = TransportFormat.MPEG2\n          }\n          this._realResource = res\n          this.done()\n        }\n      } else {\n        this.done()\n      }\n    },\n\n    _parseWithManifest: function (resp, prevParent, matches) {\n      this._lastManifest = resp\n      this.parse(resp.getResponseText(), null, prevParent + matches[2])\n    },\n\n    shouldExecute: function (lastManifest) {\n      var ret = true\n      if (lastManifest) {\n        var response = lastManifest.getResponseText()\n        ret = response.indexOf('#EXTM3U') !== -1\n      }\n      return ret\n    }\n  }\n)\n\nmodule.exports = HlsParser\n\n\n/***/ }),\n/* 26 */\n/***/ (function(module, exports, __webpack_require__) {\n\nvar Emitter = __webpack_require__(5)\nvar Log = __webpack_require__(1)\nvar Util = __webpack_require__(2)\nvar YBRequest = __webpack_require__(4)\n\nvar CdnParser = Emitter.extend(\n  /** @lends youbora.CdnParser.prototype */\n  {\n    /**\n         * Class that asynchronously tries to get information about the CDN where a given resource is\n         * hosted.\n         *\n         * The info we care about is the CDN code itself, the node host and node type.\n         *\n         * The CDN is queried with http HEAD requests. This only will work if the CDN has been properly\n         * configured.\n         *\n         * When HEAD requests are performed against the resources, the CDN returns a set of headers that\n         * contain info about the cdn header and/or cdn type.\n         *\n         * Each CDN is different; some require special headers to be set when the HEAD request is\n         * performed and others don't. Also, the info can come back in any fashion of ways, sometimes\n         * both type and host come in the same response header while sometimes they're in different\n         * headers. The format of these response headers is also different from CDN to CDN, so a\n         * different regex is used for each CDN.\n         *\n         * Lastly, as the values indicating the CDN t
```

Comment:
we do not keep the dist in the repo, we publish it into NPM on build in our CICD.
It is also excluded in the gitignore, how were you able to push it?

Reply:
I didn't see that being published neither, will remove for the next commit